CHANGELOG.md

```diff
@@ -32,10 +89,10 @@ Tag name: v0.10.2
 ## [0.10.1] - 2017-04-19
 Tag name: v0.10.1
 
-## Added
+### Added
 * Documentation for the BigchainDB settings `wsserver.host` and `wsserver.port`. [Pull Request #1408](https://github.com/bigchaindb/bigchaindb/pull/1408)
 
-## Fixed
+### Fixed
 * Fixed `Dockerfile`, which was failing to build. It now starts `FROM python:3.6` (instead of `FROM ubuntu:xenial`). [Pull Request #1410](https://github.com/bigchaindb/bigchaindb/pull/1410)
```

Comment:
I would reword this as:
Fixed `Dockerfile`, which was failing to build. 
It is based on the `python:3.6` image, rather than the `ubuntu:xenial` image.